client/packages/common/src/ui/forms/JsonForms/api/api.ts

```diff
@@ -0,0 +1,16 @@
+import { DocumentFragment, Sdk } from './operations.generated';
+
+export const getDocumentQueries = (sdk: Sdk, storeId: string) => ({
+  get: {
+    byDocName: async (name: string): Promise<DocumentFragment> => {
+      const result = await sdk.documentByName({ name, storeId });
+      const document = result?.document;
+
+      if (document?.__typename === 'DocumentNode') {
+        return document;
+      } else {
+        throw new Error('Error querying document');
+      }
```

Comment:
:nit the else isn't required and differs from the pattern elsewhere

```suggestion
      if (document?.__typename === 'DocumentNode') {
        return document;
      }

      throw new Error('Error querying document');      
```